src/atoms/LogSumExpAtom.jl

```diff
@@ -28,22 +37,29 @@ curvature(::LogSumExpAtom) = ConvexVexity()
 function evaluate(x::LogSumExpAtom)
     _x = evaluate(x.children[1])
     max_x = maximum(_x)
-    return max_x + log(sum(exp.(_x .- max_x)))
+    return max_x .+ log.(sum(exp.(_x .- max_x); x.dims))
```

Comment:
```suggestion
    return output(max_x .+ log.(sum(exp.(_x .- max_x); x.dims)))
```
We may want this so we don’t return vectors for scalars

Reply:
I think this just works. I'd already added a test for the scalar case:
```julia
@test evaluate(logsumexp(x; dims = :)) ≈ 6.456193316018123
```